README.md

```diff
@@ -15,8 +15,12 @@ By using the Beta, you hereby acknowledge that the Beta is provided "as is" with
 
 ## About this project
 
-The Mobile Core represents the core Adobe Experience Platform SDK that is required for every app implementation. The core contains a common set of functionality and frameworks, such as  Experience Cloud Identity services, data event hub, Rules Engine, reusable networking, disk access routines, and so on, which are required by all Adobe and third-party extensions.
+`AEPCore`, `AEPService`, `AEPIdentity` represent the core Adobe Experience Platform SDK extensions that are required for every app implementation. The core extensions contain a common set of functionality and frameworks, such as  Experience Cloud Identity services, data event hub, reusable networking, disk access routines, and so on, which are required by all Adobe and third-party extensions.
```

Comment:
@sbenedicadb do you want to provide better wordings for this segment?

Reply:
The `AEPCore`, `AEPServices`, and `AEPIdentity` extensions represent the foundation of the Adobe Experience Platform SDK - every app using the SDK must include them. These modules contain a common set of functionality and services which are required by all SDK extensions.

`AEPCore` contains implementation of the Event Hub. The Event Hub is the mechanism used for delivering events between the app and the SDK. The Event Hub is also used for sharing data between extensions.

`AEPServices` provides several reusable implementations needed for platform support, including networking, disk access, and database management.

`AEPIdentity` implements the integration with Adobe Experience Platform Identity services.